BinariesCheck.py

```diff
@@ -145,6 +169,26 @@ def __init__(self, pkg, path, file, is_ar, is_shlib):
                         fork_called = True
                         continue
 
+            # check if we don't have a string that will automatically
+            # waive the presence of a forbidden call
+            if self.forbidden_calls:
+                cmd = ['env', 'LC_ALL=C', 'strings']
+                cmd.append(path)
+                res = Pkg.getstatusoutput(cmd)
+                if not res[0]:
+                    for l in res[1].splitlines():
+                        # as we need to remove elements, iterate backwards
+                        for i in range(len(self.forbidden_calls)-1, -1, -1):
+                            func = self.forbidden_calls[i]
+                            try:
+                                f = BinaryInfo.forbidden_functions[func]
+                                r = f['waiver_regex'].search(l)
+                                if r:
+                                    del self.forbidden_calls[i]
```

Comment:
Please check for existence of waiver_regex before invoking it instead of using try...except here.